.changeset/cold-crabs-cross.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
No need for a changeset

Reply:
AFAIR one of jobs failed. I assume that is because the code changed is in the component's source code